src/Test/AbstractSchemaResetter.php

```diff
@@ -24,4 +24,15 @@ protected function runCommand(Application $application, string $command, array $
             throw new \RuntimeException(\sprintf('Error running "%s": %s', $command, $output->fetch()));
         }
     }
+
+    /**
+     * @param list<string> $availableObjectsToReset
+     * @param list<string> $candidateObjectsToReset
+     */
+    protected static function validateObjectsToReset(string $objectsType, array $availableObjectsToReset, array $candidateObjectsToReset): void
```

Comment:
this was made to make the code DRY, but I don't like the wording :shrug:

Reply:
Again, internal, we can refactor later.